model_deploy/templates/detection/index.html

```diff
@@ -88,73 +115,75 @@
             const userId = responseData.predictionResult.UserID;
 
             if (userId === "unknown") {
-              console.log("DATA UNKNOWN");
+              //console.log("DATA UNKNOWN");
+              showModal(-2, "", "", "Data Tidak Ditemukan!", confidence);
               // showModal(userId, confidence);
             } else {
-              try {
-                const apiResponse = await fetch("https://2108-103-243-178-32.ngrok-free.app/api/presences/ml-result", {
-                  method: "POST",
-                  headers: {
-                    "Content-Type": "application/json",
-                  },
-                  body: JSON.stringify({
-                    confidence: confidence,  // Replace with your actual confidence value
-                    userId: userId  // Replace with your actual userId
-                  }),
-                });
-
-                if (!apiResponse.ok) {
-                  throw new Error(`HTTP error! Status: ${apiResponse.status}`);
-                }
-
-                const apiResponseData = await apiResponse.json();
-                console.log("API Response:", apiResponseData);
-
-                // Display the result in a modal
-                // if (
-                //   apiResponseData &&
-                //   apiResponseData.type &&
-                //   apiResponseData.fullName &&
-                //   apiResponseData.message
-                // ) {
-                //   showModal(
-                //     apiResponseData.type, //masuk keluar 0 = masuk, 1 = pulang, 2=TidakMasukAtauPulang
-                //     apiResponseData.fullName, //nama
-                //     confidence, //confidence
-                //     apiResponseData.message // message
-                //   );
-                // }
-
-              } catch (error) {
-                console.error("Failed to connect to the API:", error);
-              }
+              //disini masukin try catch tembak API BE
+              showModal(
+                responseDummy.type,
+                responseDummy.createdAt,
+                responseDummy.fullName,
+                responseDummy.message,
+                confidence 
+              );
             }
           }
         } catch (error) {
           console.error("Error fetching data:", error);
```

Comment:
Need to add a modal error display if an error occurs in the system
```suggestion
        } catch (error) {
          showModal(-3, "", "", "", confidence);
          console.error("Error fetching data:", error);
```